Source/Core/Common/Analytics.h

```diff
@@ -59,19 +59,16 @@ class AnalyticsReportBuilder
   AnalyticsReportBuilder();
   ~AnalyticsReportBuilder() = default;
 
-  AnalyticsReportBuilder(const AnalyticsReportBuilder& other) { *this = other; }
-  AnalyticsReportBuilder(AnalyticsReportBuilder&& other)
-  {
-    std::lock_guard lk{other.m_lock};
-    m_report = std::move(other.m_report);
-  }
+  AnalyticsReportBuilder(const AnalyticsReportBuilder& other) : m_report{other.Get()} {}
+  AnalyticsReportBuilder(AnalyticsReportBuilder&& other) : m_report{other.Consume()} {}
 
   const AnalyticsReportBuilder& operator=(const AnalyticsReportBuilder& other)
   {
     if (this != &other)
     {
-      std::scoped_lock lk{m_lock, other.m_lock};
-      m_report = other.m_report;
+      std::string other_report = other.Get();
+      std::scoped_lock lk{m_lock};
+      m_report = std::move(other_report);
```

Comment:
If you only lock one mutex it probably makes more sense to use lock_guard instead of scoped_lock.